### ISSUE TITLE
When `--backtrace` is on and `Time.parse` is used to dynamically generate the name of an example, `autotest` is unable to re-run the failing specs because the first argument to `rspec` is malformed.

### ISSUE THREAD

**Issue by armandofox** [seattlerb/zentest#60]
This is the weirdest rspec/autotest bug I have seen. I'm not sure if the bug is in autotest or in part of rspec; it's beyond my RSpec-fu to narrow it down that far.  I had [opened this as an issue on rspec-dev](https://github.com/rspec/rspec-dev/issues/74), but @JonRowe looked into it and believes it's an autotest issue.

A minimal and heavily documented test case to reproduce it is at http://github.com/armandofox/rspec-backtrace

In a nutshell:
- When I try to dynamically generate the name of an example for 'specify' by calling Time.parse, and that example fails, autotest is unable to successfully re-run rspec because it mangles the command line arguments
- BUT, if the --backtrace option is removed from .rspec, the problem goes away
- OR, if the call to Time.parse is removed from the spec file, the problem also goes away.
